README.md

```diff
@@ -1,32 +1,39 @@
-# Rector - Upgrade Your Legacy App to a Modern Codebase
+# Rector
 
-Rector is a **rec**onstruc**tor** tool - it does **instant upgrades** and **instant refactoring** of your code.
-Why refactor manually if Rector can handle 80% for you?
+## What is it?
+Rector is a command line tool that can upgrade and refactor your PHP code from legacy code to modern, clean code **instantly**. 
+<br>
+
+## How can rector help you?
+- You want to upgrade from PHP 5.3 to 7.4?  Well, Rector can do that for you.  
+- You want to switch frameworks? Well, Rector can do that for you as well. 
+<br>
 
+## Want to see?
 [![Build Status](https://img.shields.io/travis/rectorphp/rector/master.svg?style=flat-square)](https://travis-ci.org/rectorphp/rector)
 [![Coverage Status](https://img.shields.io/coveralls/rectorphp/rector/master.svg?style=flat-square)](https://coveralls.io/github/rectorphp/rector?branch=master)
 [![Downloads](https://img.shields.io/packagist/dt/rector/rector.svg?style=flat-square)](https://packagist.org/packages/rector/rector)
```

Comment:
Badges should be right under the headline, it's convention

Reply:
👍